Source/m_menu.c

```diff
@@ -6471,7 +6488,6 @@ void M_ResetSetupMenu(void)
     enem_settings1[13].m_flags |= S_DISABLE;
 
   FLAG_SET_VANILLA(enem_settings1[enem_remember].m_flags, S_DISABLE);
-  FLAG_SET_VANILLA(weap_settings1[weap_recoil].m_flags, S_DISABLE);
```

Comment:
doesn't this have to be restored now?

Reply:
Fixed in https://github.com/fabiangreffrath/woof/pull/506/commits/afa3b1b44796848aae8ed53509aea8e20a0b2248